[msal-node-extensions]: Enhance the msal-node-extensions sample

samuelkubai opened the pull request:
- add a cache route to the extensions sample to access the cache
- enhance the redirect page with useful links and information

Commit:
feat(extensions-sample): enhance the msal-node-extensions sample

- add a cache route to the extensions sample to access the cache
- enhance the redirect page with useful links and information

Commit:
feat(msal-node-extensions): add benchmark times to the various clients

jasonnutter commented on extensions/msal-node-extensions/src/persistence/FilePersistenceWithDataProtection.ts:
```diff
@@ -39,28 +39,39 @@ export class FilePersistenceWithDataProtection implements IPersistence {
                 this.scope.toString());
             await this.filePersistence.saveBuffer(encryptedContents);
+            // eslint-disable-next-line no-console
+            console.timeEnd(TIMESTAMP_LABEL);
```
I am curious about the need to include these timings in the extension library itself, can you provide some details on that?

samuelkubai replied:
I added the timings temporarily to be able to get the timings across the various environments easily. I don't plan to leave this piece of code there.

samuelkubai replied:
Also converted the pull request to a draft till I clean it up.

samuelkubai committed:
Merge branch 'dev' into node-extensions-e2e

Commit:
feat: add ability to run multiple applications simultaneously

Commit:
Merge branch 'node-extensions-e2e' of github.com:AzureAD/microsoft-authentication-library-for-js into node-extensions-e2e

Commit:
feat: add support for multiple applications in node-extensions sample

- add multiple application configuration in the node-extensions sample
- add the server port to the root page in the node-extensions sample
